frontend/src/views/RouteFormView.vue

```diff
@@ -92,6 +92,8 @@
                 <div class="form-group col-sm-6">
                   <label>{{ $t('routes.starting_mileage') }}</label>
                   <input
+                    min="0"
+                    max="1500000"
```

Comment:
Maybe they have a reeeealy good diesel engines and they can go further than that? 😃

Reply:
Due to curiosity I checked some ridiculously long routes and e.g. there is 13 632 km from Porto to Wladywostok :P In 1500000 you could circle the earth almost 38 times ^^